pages/documentation/play.md

```diff
@@ -6,4 +6,6 @@ weight: 5
 
 To start automation user basically need to select the project and click "play" button. After clicking "play" button actions begin to be injected to content script one by one, and you can follow the workflow process in actions datagrid, also you can see "play" message appear under the CBA icon that mean that Chromium Browser Automation is in the process of playing your scenarios, So in case if the workflow will stuck somewhere you always can click on "stop" button. "stop" Button stops the project workflow. Also user can mention how many times he want project to be played by changing the value in the textbox next to "play" button. 
 
```

Comment:
Here we have unnecessary paragraph I guess.

Reply:
AFAIK, when you have a block level element proceeding the new line, then an empty wrapper won't be generated, unless you add a content there.